lib/exercises.rb

```diff
@@ -1,23 +1,51 @@
-
-
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
-
+# Time Complexity: O(n)
+# Space Complexity: O(n^2)
+strings = ["eat", "tea", "tan", "ate", "nat", "bat"]
 def grouped_anagrams(strings)
```

Comment:
Why O(n^2) space complexity?  You're making a new hash, but that doesn't take that much space...

Otherwise this works.